### PR TITLE
[SP] Check the returned volume preview

### PR DESCRIPTION
**Pull request by DonnaWuDongxia**
Unchecking the returned volume preview image may cause extension process interrupt with out range camera pose.

### CODE REVIEW

File: realsense/scene_perception/win/scene_perception_object.cc [hunk 1]
```diff
@@ -1432,6 +1432,12 @@ void ScenePerceptionObject::DoQueryVolumePreview(
   }
 
   PXCImage* vPreview = scene_perception_->QueryVolumePreview(pose);
```
Comment (huningxin) [inline realsense/scene_perception/win/scene_perception_object.cc]: Please use variable_name style, such as "preview_image".

File: realsense/scene_perception/win/scene_perception_object.cc [hunk 2]
```diff
@@ -1432,6 +1432,12 @@ void ScenePerceptionObject::DoQueryVolumePreview(
...
+  if (!vPreview) {
+    info->PostResult(
+        QueryVolumePreview::Results::Create(
+          image, "Failed to queryVolumePreview"));
```
Comment (huningxin) [inline realsense/scene_perception/win/scene_perception_object.cc]: "Failed to execute queryVolumePreview"?

**Comment by huningxin** [comment]
LGTM with nits. Please take a look.

File: realsense/scene_perception/win/scene_perception_object.cc (hunk 2 above)
Comment (DonnaWuDongxia) [inline realsense/scene_perception/win/scene_perception_object.cc]: Yes, similar case. It may return null if the pose parameter was not suitable.

**Comment by DonnaWuDongxia** [comment]
@huningxin updated, thanks!

File: realsense/scene_perception/win/scene_perception_object.cc [hunk 3]
```diff
@@ -1431,11 +1431,17 @@ void ScenePerceptionObject::DoQueryVolumePreview(
     pose[i] = static_cast<float>((params->pose)[i]);
   }
 
-  PXCImage* vPreview = scene_perception_->QueryVolumePreview(pose);
+  PXCImage* volume_preview = scene_perception_->QueryVolumePreview(pose);
+  if (!volume_preview) {
+    info->PostResult(
+        QueryVolumePreview::Results::Create(
+          image, "Failed to queryVolumePreview"));
```
Comment (huningxin) [inline realsense/scene_perception/win/scene_perception_object.cc]: This line should be 4-spaces indent.

**Comment by huningxin** [comment]
@DonnaWuDongxia , another nit. Sorry not pointing it in the first time.

**Comment by DonnaWuDongxia** [comment]
@huningxin updated, thanks!

**Comment by huningxin** [comment]
Great! Thanks much.

LGTM. Merging.

**Pull request merged**